docs/wiki/Client.mediawiki

```diff
@@ -1,4 +1,63 @@
-==Client (dexc) ==
+=Guides=
```

Comment:
Am I the only one who seriously dislikes mediawiki?  Would anything be lost by using md?  I guess the nice tables for the various payloads rely on it.

BTW, what's the audience for this page?  Before it was some advanced technical docs, which are still at the bottom, but now it begins with some basic setup stuff.

I hope my modifications to the README "quick start" in https://github.com/decred/dcrdex/pull/1747 aren't off-base, but I feel like I've reduced it to very basics that a prospective user dropping in on the github repo would need.

Reply:
> Am I the only one who seriously dislikes mediawiki? Would anything be lost by using md? I guess the nice tables for the various payloads rely on it.

Meh.

> BTW, what's the audience for this page? Before it was some advanced technical docs, which are still at the bottom, but now it begins with some basic setup stuff.

I was kind of thinking that too. The page seems to be a catchall for client stuff. I figured we can always break it up later.

Reply:
> I figured we can always break it up later

Sounds good.  Let's just get things moving.  It's just the wiki.

Reply:
Do... you want to do .md? I have absolutely no preference and don't know a whole lot about either.

Reply:
Ok to leave this client control mediawiki since it actually uses mediawiki stuff, but might as well keep things markdown if they are already markdown, and new pages should prefer md.

Reply:
Newest comment just makes a new `.md` file.